packages/dapp/src/components/DualWidget/DualWidget.tsx

```diff
@@ -56,35 +65,90 @@ export function DualWidget() {
     }
   }, [activeTab, onChangeTab, starterVariant, starterVariantUsed]);
 
+  const handleGetStarted = async () => {
+    setShowFadeOut(true);
+    await setTimeout(() => {
+      setShowWelcome(false);
+    }, 300);
+  };
+
+  const isWalletConnected = useMemo(() => {
+    const activeWallet = JSON.parse(
+      localStorage.getItem(LOCAL_STORAGE_WALLETS_KEY),
+    );
+    return activeWallet?.length > 0 || false;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [account.isActive]);
```

Comment:
@chybisov Actively disabling the linter on here - I want this to be re-loaded whenever there is a change in the account. Although linter tells me to remove it from the dependencies (because it is not used within the useMemo), I keep it. Is that a correct approach?